webpack/assets/javascripts/react_app/components/SearchButton/SearchButton.js

```diff
@@ -0,0 +1,19 @@
+import URI from 'urijs';
+import React from 'react';
+import { Button, Icon } from 'patternfly-react';
+
+const handleClick = (searchQuery) => {
```

Comment:
would it be better to handle this  in the component's consumer ? with this approach the `onClick` will be dynamic.